package.json

```diff
@@ -37,7 +37,7 @@
     "@babel/eslint-parser": "^7.21.3",
     "@babel/plugin-proposal-decorators": "^7.21.0",
     "@ember/optional-features": "^2.0.0",
-    "@ember/string": "^3.0.1",
+    "@ember/string": "^3.1.1",
```

Comment:
Had to keep the ember/string devDependency, as ember-resolver 10 has a peerDependency for it

This will have no affect on downstream users